### PR TITLE
chore: update scriptappy

### PR DESCRIPTION
**Pull request by niekvanstaveren**

### CODE REVIEW

File: spec-configs/props.conf.js [hunk 1]
```diff
@@ -3,24 +3,28 @@ const path = require("path");
...
-  },
-  parse: {
-    types: {
-      "qae.GenericObjectProperties": {},
+    output: {
+      file: path.resolve(__dirname, "../api-specifications/properties.json"),
+    },
+    parse: {
+      types: {
+        GenericObjectProperties: {
```
Comment (Caele) [inline spec-configs/props.conf.js:24]:
```suggestion
        EngineAPI.GenericObjectProperties: {
```

File: src/object-properties.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import DEFAULTS from "./style-defaults";
 
 /**
- * @extends {qae.GenericObjectProperties}
+ * @extends {GenericObjectProperties}
```
Comment (Caele) [inline src/object-properties.js:4]:
```suggestion
 * @extends {EngineAPI.GenericObjectProperties}
```

File: spec-configs/props.conf.js (hunk 1 above)
Comment (Caele) [reply spec-configs/props.conf.js:24]: + update in the props file and run it again

Comment (niekvanstaveren) [reply spec-configs/props.conf.js:24]: But the we should also reference it like that here https://github.com/qlik-oss/sn-action-button/pull/428

Comment (niekvanstaveren) [reply spec-configs/props.conf.js:24]: Also how wordcloud actually references it https://github.com/qlik-oss/sn-word-cloud/blob/main/api-spec/props.conf.js#L26

File: package.json [hunk 3]
```diff
@@ -89,6 +89,8 @@
     "@playwright/test": "1.38.0",
     "@qlik/eslint-config": "0.4.18",
     "@qlik/prettier-config": "0.3.0",
+    "@scriptappy/cli": "^0.10.0",
```
Comment (cbt1) [inline package.json:92]: Should pin the dependencies

**Review by cbt1 — APPROVED** [review]
Cannot really comment on how valid the spec changes are but it LGTM

**Comment by niekvanstaveren** [comment]
> Cannot really comment on how valid the spec changes are but it LGTM

No it is quite a diff. Tried to compare it and it looks mostly like re-ordering

**Pull request merged**